plugin/storage/es/spanstore/dbmodel/model.go

```diff
@@ -88,3 +88,9 @@ type Service struct {
 	ServiceName   string `json:"serviceName"`
 	OperationName string `json:"operationName"`
 }
+
+// ServiceWithKind is the JSON struct service:kind:operation documents in ElasticSearch
+type ServiceWithKind struct {
+	Service
+	Kind string `json:"spanKind"`
```

Comment:
not sure I follow. Why wouldn't we just add Kind field to the Service struct above?

Reply:
Please see the comment. As then in query for fetching spans of all kinds, we have apply another filter with kind as empty.